relationships/synthesis/INFRA-KAFKABROKER-to-INFRA-HOST.stg.yml

```diff
@@ -0,0 +1,59 @@
+relationships:
```

Comment:
same here. if you want a stg file you will need to provide all rules or they will not be created